Assets/Scripts/EnemyController.cs

```diff
@@ -2,9 +2,11 @@
 using System.Collections.Generic;
 using UnityEngine;
 
+public enum MovementType { PATH, MONGO };
+
 public class EnemyController : MonoBehaviour
 {
-    public int iD;
```

Comment:
I know you probably did this because you wanted the editor to show "ID" instead of "Id", but I would personally pick code standardization over it looking slightly nicer in the Unity editor.